packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -998,7 +1041,8 @@ describe("wrangler", () => {
                 --binding       The name of the namespace to get from  [string]
                 --namespace-id  The id of the namespace to get from  [string]
             -e, --env           Perform on a specific environment  [string]
-                --preview       Interact with a preview namespace  [boolean] [default: false]"
+                --preview       Interact with a preview namespace  [boolean] [default: false]
+                --text          Encode the response as UTF8  [boolean] [default: false]"
```

Comment:
```suggestion
                --text          Decode the returned value as a utf8 string  [boolean] [default: false]"
```

Reply:
This should have updated with snapshot updates...?